journal/profiles.cpp

```diff
@@ -596,6 +607,8 @@ void SetPlottingFrame::Run(Message const& message,
   auto* navigation_frame = DeserializePointer<NavigationFrame*>(
                                *pointer_map, in.navigation_frame());
   ksp_plugin::principia__SetPlottingFrame(plugin, &navigation_frame);
+  // TODO(phl): This may delete the previous plotting frame, which should thus
+  // be removed from |pointer_map|.
```

Comment:
No, the `pointer_map` doesn't represent the memory (we don't care about the memory), it represents the objects that the caller of the DLL owns directly. Thus `SetPlottingFrame` "deletes" (consumes) *its argument*.